### ISSUE TITLE
View existing datapipeline

### ISSUE THREAD

**Issue by Elementator** [amosproj/amos2023ws04-pipeline-manager#100]
## User story
1. As a User
2. I want to see all the datapipelines available in this project
3. So that I can chose from them for processing my files

## Acceptance criteria

- [ ] Frontend shows all datapipelines currently available in the project
- [ ] Backend service that provides an overview of all available datapipelines

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been tested
* Feature has been merged into the main branch
* Important parts have been documented in the technology wiki
* All acceptance criteria were met
* Product owner approved features
* Developers agreed to release
